src/test/java/io/dropwizard/jdbi/unitofwork/core/LinkedRequestScopedJdbiHandleManagerTest.java

```diff
@@ -0,0 +1,114 @@
+package io.dropwizard.jdbi.unitofwork.core;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.stubbing.Answer;
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.Handle;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class LinkedRequestScopedJdbiHandleManagerTest {
+
+    private DBI dbi;
+
+    private LinkedRequestScopedJdbiHandleManager manager;
```

Comment:
This class is all about testing whether multiple threads can share a common thread handle safely